Content.Server/DropPod/DropPodConsoleComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Audio;
+
+namespace Content.Server.DropPod
+{
+    [RegisterComponent]
+    public sealed partial class DropPodConsoleComponent : Component
+    {
+        public bool WarDeclared = false; // This field is needed to prevent players from flying away on a capsule before the declaration of war
+
+        [DataField("announcement")]
+        public bool Announcement = true;
+
+        [DataField("text")]
```

Comment:
```suggestion
        [DataField]
```
also rename variable, something like `AnnouncementText`